### PR TITLE
Release: 2022-06-30

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

New release candidate.

# Changes

* Changelog.

# Tests

Not applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+* Add light theme
+* Enable theming
+* Upgrade agent-js
+* Improve signout flow
+* Improvements in error management
+
+
 ## Proposal 66831
 * Update canister payment flow
 * New Navigation menu
```
Comment (peterpeterparker) [inline CHANGELOG.md:13]: - Lazy load routes to improve performance

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Proposal XXXXX
+* Add light theme
+* Enable theming
+* Upgrade agent-js
+* Improve signout flow
+* Improvements in error management
+
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply CHANGELOG.md]: done, thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Proposal XXXXX
+* Add light theme
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply CHANGELOG.md:4]: done thanks

**Comment by lmuntaner** [comment]
@bitdivine do we need anything else before merging this one?

**Review by peterpeterparker — APPROVED** [review]
Thanks

**Pull request merged**